nslord/Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM ubuntu:bionic
 
 RUN apt-get update && apt-get install -y \
-		dnsutils \
-		net-tools \
-		dirmngr gnupg \
-		faketime \
-	--no-install-recommends && apt-get clean && rm -rf /var/lib/apt/lists/*
+    dnsutils \
+    iptables \
```

Comment:
I don't think that's the right approach. We should first install the common stuff, and further down the non-common stuff, instead of installing everything everywhere.

Also, multi-stage builds may help (but I'm not sure).

Reply:
This deserves some more discussion. I've excluded this commit from this PR and moved it to #633.